### PR TITLE
fix(chat): fix installed models init and install recent models

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

fix installed models init

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2181.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2181.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/utils/app/conversation.ts [hunk 1]
```diff
@@ -215,9 +211,6 @@ export const isChosenConversationValidForCompare = (
   return true;
 };
 
-export const getOpenAIEntityFullName = (model: DialAIEntity) =>
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/conversation.ts:218]: why did you remove this method and made a lot of changes instead of just modify it?

**Review by IlyaBondar — APPROVED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2181.nightly-test.deltixhub.io
>E2E tests status: failed
